### PR TITLE
shorten configuration code

### PR DESCRIPTION
**Pull request by PabloRuizCuevas**
## Summary

Solve the issue: #___

## Quick changelog

- <Reduce verbosity>

## What's new?

Remade pr

### CODE REVIEW

File: freqtrade/configuration/configuration.py [hunk 1]
```diff
@@ -68,10 +68,8 @@ def load_config(self) -> Dict[str, Any]:
         config: Config = load_from_files(self.args.get("config", []))
 
         # Load environment variables
-        from freqtrade.commands.arguments import NO_CONF_ALLOWED
-        if self.args.get('command') not in NO_CONF_ALLOWED:
-            env_data = enironment_vars_to_dict()
-            config = deep_merge_dicts(env_data, config)
+        env_data = enironment_vars_to_dict()
+        config = deep_merge_dicts(env_data, config)
```
Comment (xmatthias) [inline freqtrade/configuration/configuration.py]: We'll definitely want to keep this conditional.

it's intentionally there (added recently) - protecting some odd scenarios where environment variables mess with commands that shouldn't load a configuration.

**Review by xmatthias — CHANGES_REQUESTED** [review]
There's one too  many changes (see other comment).

---

For the future, please do Pull requests from a different Branch (not your main branch).

An explanation as to why you should use feature branches can be found [here](https://hynek.me/articles/pull-requests-branch/).

keep this PR as is though (this is for an eventual next time ...) - you can update this PR by simply committing and pushing to this branch again.

File: freqtrade/configuration/configuration.py (hunk 1 above)
Comment (PabloRuizCuevas) [reply freqtrade/configuration/configuration.py]: Should be solved, thanks for the explanation.

**Review by xmatthias — APPROVED** [review]
LGTM 👍

**Pull request merged**